includes/html/pages/device/port/macaccounting.inc.php

```diff
@@ -47,32 +47,32 @@
            <div style='margin: 0px 10px 5px 0px; padding:5px; background: #e5e5e5;'>
            <span class=device-head>Day</span><br />
 
-           <a href='".generate_url($link_array, array('view' => 'macaccounting', 'subview' => 'top10', 'graph' => $vars['graph'], 'sort' => $vars['sort'], 'period' => 'day'))."'>
+           <a href='".generate_url($link_array, array('view' => 'macaccounting', 'subview' => 'top10', 'graph' => $vars['graph'], 'sort' => $vars['sort'], 'period' => '1d'))."'>
```

Comment:
These dates don't work.  I think it should be a timestamp.

Reply:
Can you be more specific regarding of what exactly doesn't work, please? According to [RRDtool documentation](https://oss.oetiker.ch/rrdtool/doc/rrdfetch.en.html#AT-STYLE_TIME_SPECIFICATION), time offset units can be specified as abbreviations too. Generated RRDtool command looks like this:

`rrdtool graph /tmp/b6L3kMK3W97WqQp5  --alt-autoscale-max --rigid -E --start -1d --end 1558349100 --width 745 --height 300 -c BACK#EEEEEE00 -c SHADEA#EEEEEE00 ... <additional output omitted>`

...and works like a charm. I use RRDtool v1.6.0.